vendor-bin/symfony/composer.json

```diff
@@ -5,9 +5,16 @@
         ]
     },
     "require-dev": {
+        "alcaeus/mongo-php-adapter": "^1.0.10",
         "doctrine/data-fixtures": "^1.2",
         "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/mongodb": "^1.4",
+        "doctrine/mongodb-odm-bundle": "^3.0",
         "doctrine/orm": "^2.5",
+        "doctrine/phpcr-bundle": "^1.0",
+        "doctrine/phpcr-odm": "^1.4",
+        "jackalope/jackalope-doctrine-dbal": "^1.2",
+        "psy/psysh": "^0.6",
```

Comment:
Yeah. It brings too many deps so the difference between the deps the consumer will get and ours during tests will be too great